index.html

```diff
@@ -0,0 +1,134 @@
+    <link href="https://fonts.googleapis.com/css2?family=Geologica:wght@300;400;600;700;900&family=Lato:wght@300&display=swap" rel="stylesheet">
+</head>
+<header>
+
+    <body>
+
+        <nav class="icon-nav">
+            <ul class="for-icon-nav">
+                <li>
+                    <a href="#"><img src="images/fb.png" alt="blue-mark"></a>
+                </li>
+                <li>
+                    <a href="#"><img src="images/twitter.png" alt="blue-bird"></a>
+                </li>
+                <li>
+                    <a href="#">English</a>
+                </li>
+                <li>
+                    <a href="#">My Page</a>
+                </li>
+                <li>
+                    <a href="#">Log out</a>
+                </li>
+            </ul>
+
+        </nav>
+</header>
+<div class="container-two">
+    <nav class="lower-nav">
+        <img class="music-logo" src="images/piano logo.png" alt="logo">
+        <ul class="for-desk-nav" id="for-nav">
+            <li><a href="aboutpage.html" id=" ">About</a></li>
+            <li>
+                <a href="#">Program</a></li>
+            <li><a href="#">Tickets</a></li>
+            <li><a href="#">Sponsors</a></li>
+            <li><button class="for-border">Summer concert</button></li>
+        </ul>
+    </nav>
+
+    <section class="concert-introduction">
+        <div class="to-move">
+            <img src="/images/hamburger.svg" alt="" id="menu-two" onclick="toggleMenu()">
+            <nav id="for-hamburger" class="menu-hidden">
+                <ul class="for-style">
+                    <li>
+                        <img class="for-close" src="/images/close-button.png" alt="close-button" id="for-close-burger">
+                    </li>
+                    <li><a href="aboutpage.html" id=" ">About</a></li>
+                    <li>
+                        <a href="#">Program</a></li>
+                    <li><a href="#">Tickets</a></li>
+                    <li><a href="#">Sponsors</a></li>
+
+                </ul>
+            </nav>
+            <h4 class="first-content">Hello Piano World!</h4>
+            <div class="content">
+                <h1>PIANO CONCERT</h1>
+                <h1>GLOBAL SUMMIT</h1>
+                <h1>2023</h1>
+            </div>
+            <div class="lorem-border">
+                <p>
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolorum tempore aliquid est minus odit vero facere necessitatibus voluptatem blanditiis, perferendis ducimus quo, cum fugit eligendi, sint vitae officiis. Nostrum cum sit quam consequuntur vel modi
+                    nam distinctio architecto earum praesentium repellendus mollitia voluptatum voluptate cupidit
+                </p>
+            </div>
+            <p class="concert-time">28 July 2023 from 10:00 PM</p>
+            <p class="concert-time-sec">@National Museum of Myanmar Art Center Man and more</p>
+        </div>
+</div>
+</section>

```

Comment:
- [ ] Well done on the improvements so far. However, we still need to ensure that the concerns from the previous review are duly satisfied. We still need to improve on the design as mentioned by the previous reviewer. I will highlight the areas that we need to work on below:

|The Provided Design | Your implementation |
| --- | --- |
| ![image](https://github.com/Hsuwaddyhtat22/musical-giggle-capstone/assets/45002654/9436b5db-18bd-4740-a012-c533b1b86555) | ![hero](https://github.com/Hsuwaddyhtat22/musical-giggle-capstone/assets/45002654/16dbe301-dad0-43eb-b259-834302abbf2b) |

As evident in the comparison above. There is still a lot to be done. Please consider the following:

- Margins: There are equal margins on both sides of the page. Please align the content of the page as in the provided design.
- Font Weight: Where the design uses a bolder font, please align with the same design.